### ISSUE TITLE
`stripe login` failed right after installation

### ISSUE THREAD

**Issue by bz-stripe** [stripe/stripe-cli#182]
stripe version 0.6.5 (beta)

### Issue
Ran `stripe login` and received error message

```
Post https://dashboard.stripe.com/stripecli/auth: http2: server sent GOAWAY and closed the connection; LastStreamID=1, ErrCode=ENHANCE_YOUR_CALM, debug=""
```

### Expected Behavior
According to the readme: "You'll be redirected to the Stripe dashboard to confirm that you want to give access to your account to the CLI. After confirming, a new API key will be created and returned to the CLI."

### Steps to reproduce
```
brew install stripe/stripe-cli/stripe
stripe login
```

### Traceback
```
Post https://dashboard.stripe.com/stripecli/auth: http2: server sent GOAWAY and closed the connection; LastStreamID=1, ErrCode=ENHANCE_YOUR_CALM, debug=""
```

### Environment
macOS

**Comment by ob-stripe** [comment]
@bz-stripe Thanks for the report. We've identified the issue and will release a fix soon.

**Comment by ob-stripe** [comment]
Should be fixed in 0.6.6.

**Comment by bz-stripe** [comment]
thanks!
